src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -576,7 +576,7 @@ def get_versioned_sdk_path(api_profile, resource_type, operation_group=None):
     """
     api_version = get_api_version(api_profile, resource_type)
     if api_version is None:
-        return resource_type
+        return resource_type.import_prefix
```

Comment:
The returned value should be sdk_path which is a `str` instead of `ResourceType`.

Reply:
When will `api_version` be `None`? Won't non-existing `resource_type` trigger a `APIVersionException`?

https://github.com/Azure/azure-cli/blob/e48840dba9cb419b9d0d1d451aff8f768b88c966/src/azure-cli-core/azure/cli/core/profiles/_shared.py#L392-L410

Reply:
No existing `resource_type` has `None` as it's `api_version`. But in keyvault track2 data plane, we will have the first try😥